ui/app/containers/nodeGraphs.tsx

```diff
@@ -273,6 +273,24 @@ export default class extends React.Component<IInjectedProps, {}> {
               </Axis>
             </LineGraph>
 
+            <LineGraph title="Critical Section Time"
+                       tooltip={`The maximum duration for which the corresponding mutex was held in the last minute ${specifier}.`}
+                       sources={sources}>
+              <Axis format={ (n: number) => d3.format(".1f")(NanoToMilli(n)) } label="Milliseconds">
+                <Metric name="cr.store.mutex.storenanos-max" title="StoreMu"
+                        aggregateMax downsampleMax />
+              </Axis>
+              <Axis format={ (n: number) => d3.format(".1f")(NanoToMilli(n)) } label="Milliseconds">
```

Comment:
Whoops, not sure what I did there. Certainly didn't want more than one axis. https://github.com/cockroachdb/cockroach/pull/9835